spring-ai-core/src/main/java/org/springframework/ai/model/function/FunctionCallingOptions.java

```diff
@@ -18,11 +18,12 @@
 
 import java.util.List;
 import java.util.Set;
+import org.springframework.ai.chat.prompt.ChatOptions;
 
 /**
  * @author Christian Tzolov
  */
-public interface FunctionCallingOptions {
+public interface FunctionCallingOptions extends ChatOptions {
```

Comment:
FunctionCallingOptions not extending ChatOptions was intentional in order to provide support finer grained support of what features can be opted into.  Also, the code didn't compile after this change, so be sure to do a compilation of the total project before submitting a PR please.